prow/sidecar/censor.go

```diff
@@ -65,6 +65,11 @@ func (o Options) censor() error {
 
 	secrets, err := loadSecrets(o.CensoringOptions.SecretDirectories, o.CensoringOptions.IniFilenames)
 	if err != nil {
+		// TODO(petr-muller): This return makes the censoring mechanism fragile, single failure in `loadSecrets`
+		// will prevent us from censoring all other secrets that were successfully loaded. Alternatively,
+		// we could be more strict and just bail out at our callsite in run.go:preUpload() instead of just
```

Comment:
I can follow this up with an option at the censoring config level that makes the preUpload failure case the whole process to fail if set, something opposite of "BestEffort bool`?